brainscore_language/plugin_management/environment_manager.py

```diff
@@ -0,0 +1,45 @@
+from pathlib import Path
+import subprocess
+import warnings
+
+
+class EnvironmentManager:
+    """ Runs plugins in conda environments """
+
+    def __init__(self):
+        self.envs_dir = Path(self.get_conda_base()) / 'envs'
+        self.env_name = 'brainscore-language'
```

Comment:
what if this environment already exists? (e.g. I named my dev environment exactly this)

Reply:
This name should be overwritten in every existing case, but for extra future proofing, I've appended a random int to the end